Cargo.toml

```diff
@@ -18,8 +18,9 @@ clap = "3.0.0-beta.2"
 indicatif = "0.15.0"
 log = "0.4.11"
 pretty_env_logger = "0.4.0"
-roxmltree = "0.14.0"
+roxmltree = { version = "0.14.0", features = ["std"] }
```

Comment:
`roxmltree` is now `no_std` by default, so we're turning on the `std` feature to gain access to the `std::error::Error` implementation of `roxmltree::Error`.